includes/class-newspack-popups-model.php

```diff
@@ -445,8 +445,8 @@ protected static function insert_event_tracking( $popup, $body, $element_id ) {
 								"request": "event",
 								"selector": "#<?php echo esc_attr( $element_id ); ?> <?php echo esc_attr( $mailchimp_form_selector ); ?>",
 								"extraUrlParams": {
-									"popup_id": "<?php echo ( esc_attr( $popup['id'] ) ); ?>",
-									"url": "<?php echo esc_url( home_url( $wp->request ) ); ?>",
+									"popup_id": "<?php echo self::canonize_popup_id( esc_attr( $popup['id'] ) ); ?>",
```

Comment:
In two places PHPCS asks for escaping on this line. Would it work the same if we switched the order like this?

```
"popup_id": "<?php echo esc_attr( self::canonize_popup_id( $popup['id'] ) ); ?>",
```

Reply:
Done in 856d73618a30aca8f7940edb5b4983225d675b97